<launch>
  <!-- turtlebot_teleop_key already has its own built in velocity smoother -->
  <node pkg="simulated_mirroring" type="keyboard_teleop" name="keyboard_teleop"  output="screen">
    <param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>
    <!-- <remap from="keyboard_teleop/cmd_vel" to="/turtlebot/cmd_vel_mux/input/teleop"/> -->
    <!-- <remap from="keyboard_teleop/cmd_vel" to="/simbot1/sim_cmd_vel"/> -->
    <remap from="keyboard_teleop/cmd_vel" to="cmd_vel_mux/input/teleop"/>
  </node>

</launch>
